### DISCUSSION TITLE
download had incorrect check sum.

### DISCUSSION THREAD

**Discussion by cb49747** [Homebrew/discussions#2848] [Everyday usage] [answered]
attempted to install telnet and it went to update homebrew first.  that download had a incorrect checksum.  now I cant do anything because it attempts to update say it has downloaded and download has bad checksum.  How do I get rid of that down load?

here is error.

==> Downloading https://ghcr.io/v2/homebrew/portable-ruby/portable-ruby/blobs/sha256:0cb1cc7af109437fe0e020c9f3b7b95c3c709b140bde9f991ad2c1433496dd42
Already downloaded: /Users/christopherburger/Library/Caches/Homebrew/portable-ruby-2.6.8.yosemite.bottle.tar.gz
Error: Checksum mismatch.
Expected: 0cb1cc7af109437fe0e020c9f3b7b95c3c709b140bde9f991ad2c1433496dd42
  Actual: efb57c09f137f72c1c051461107be1cc14ee1ed25b42bf5abaf084c34896519d
 Archive: /Users/christopherburger/Library/Caches/Homebrew/portable-ruby-2.6.8.yosemite.bottle.tar.gz
To retry an incomplete download, remove the file above.
Error: Failed to upgrade Homebrew Portable Ruby!

**Comment by SMillerDev** [comment, marked as answer]
Have you tried removing the file?

```
Already downloaded: /Users/christopherburger/Library/Caches/Homebrew/portable-ruby-2.6.8.yosemite.bottle.tar.gz
```

**Reply by kmraditya108** [reply]
great! it works.
@thanks @SMillerDev

**Comment by cb49747** [comment]
That did it.  was not sure I could just delete that file.  Thanks for the help
